packages/application-shell/src/components/user-settings-menu/messages.ts

```diff
@@ -16,4 +16,14 @@ export default defineMessages({
     description: 'The label for privacy policy option',
     defaultMessage: 'Privacy Policy',
   },
+  openMenuLabel: {
+    id: 'UserSettingsMenu.openMenuLabel',
+    description: 'The label when menu is closed and would be opened by a click',
+    defaultMessage: 'Open User Settings Menu',
+  },
+  closeMenuLabel: {
+    id: 'UserSettingsMenu.closeMenuLabel',
+    description: 'The label when menu is open and would be closed by a click',
+    defaultMessage: 'Close User Settings Menu',
```

Comment:
The elements are visible on screen. As a result it makes sense to also translate the label.